tools/test.py

```diff
@@ -155,9 +161,20 @@ def main():
             for ds_cfg in cfg.data.test:
                 ds_cfg.pipeline = replace_ImageToTensor(ds_cfg.pipeline)
 
+    if args.gpu_ids is not None:
+        cfg.gpu_ids = args.gpu_ids
+    else:
+        cfg.gpu_ids = range(1)
+
     # init distributed env first, since logger depends on the dist info.
     if args.launcher == 'none':
         distributed = False
+        if len(cfg.gpu_ids) > 1:
+            warnings.warn(
+                'Only supports single GPU in non-distribute testing time.'
```

Comment:
a warning message is confusing，please polish the description

Reply:
change to f'We treat {cfg.gpu_ids} as gpu-ids, and reset to {cfg.gpu_ids[0:1]} as gpu-ids to avoid potential error in non-distribute testing time.